src/app/main/component/user/components/profile/edit-profile/social-networks/social-networks.component.ts

```diff
@@ -36,17 +35,25 @@ export class SocialNetworksComponent implements ControlValueAccessor {
 
   constructor(private dialog: MatDialog) {}
 
-  onChange = (value: any) => {};
+  onChange(value: any) {
+    // TODO: add functionality to this method
+  }
 
-  writeValue(obj: any): void {}
+  writeValue(obj: any): void {
+    // TODO: add functionality to this method
+  }
```

Comment:
the same

Reply:
Are you sure about removing these methods? Though they are empty, their existence are notification that something not quite done.

Reply:
yes, remove them

Reply:
done